### DISCUSSION TITLE
Femme-coded customizations

### DISCUSSION THREAD

**Discussion by fables-tales** [BattlesnakeOfficial/feedback#103] [General]
Going through the customizations today, I noticed something, while there are only a few that could reasonably be coded as gendered at all, none (in my opinion) can be coded as femme/feminine, but a few can be easily coded as masc/masculine. I will note these are the ones I have unlocked too. I would suggest the following are clearly masculine coded:
* rbc-bowler
* rbc-necktie
* bonhomme (head + tail) (literally translating as good-man)
* villain

It'd be nice if there were some explicitly feminine coded customizations open to all as well. 

Why? Well, representation matters, and I feel like it's hard for me to express myself through snake customisations right now.

**Comment by bvanvugt** [comment]
Yes! 100%. This is already in the works, and a strict requirement for the next round of art. Would love to show you some early concepts.

**Reply by fables-tales** [reply]
❤️  for sure, feel free to pop me a DM in whichever channel makes the most sense
